pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -61,7 +61,8 @@ export const CryptoPolicyRow = () => {
                     } catch (error) {
                         console.warn("Failed to get current crypto policy:", error.toString(),
                                      "; falling back to /etc/crypto-policies/config");
-                        setCurrentCryptoPolicy(contents.trim());
+                        const filteredContent = contents?.split('\n').filter(line => !line.startsWith("#")).join('\n');
```

Comment:
contents must exist at this point, but it doesn't hurt.

Reply:
whoop, that was my autocomplete who happened to be confused by the lack of TS in a JS file